_posts/2017/2017-02-28-Using common development environmet.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "One environment to rule them all"
+date: "2017-02-22 07:00:00"
+image: blog-header/ring.jpg
+categories: tech-stories
+authors: ["Jordi Viladoms"]
+---
+
+One of the most popular problems that developers have to face is to work with a common development environment. At the same time, this environment has to be the same that the production one. ePages software is working into servers with centOS or debian and it makes mandatory for the developers to write the code under this environment. For sure make no sense to force all the developers to have one of those OS installed at the laptop taking into account that the most preferred OS are [macOS, ubuntu or windows](https://developer.epages.com/blog/2016/12/22/the-battle-of-the-operating-systems.html).
```

Comment:
1)I say that development environment has to be the same than the production environment.
2) It is an example of have the same environment at development and production. At production we have centOS and debian. So, at development we also have to work with centOS and debian.